lib/SAMLSettings.php

```diff
@@ -119,6 +119,7 @@ public function getOneLoginSettingsArray($idp) {
 				'wantXMLValidation' => ($this->config->getAppValue('user_saml', $prefix . 'security-wantXMLValidation', '0') === '1') ? true : false,
 				'requestedAuthnContext' => false,
 				'lowercaseUrlencoding' => ($this->config->getAppValue('user_saml', $prefix . 'security-lowercaseUrlencoding', '0') === '1') ? true : false,
+				'signatureAlgorithm' => $this->config->getAppValue('user_saml', $prefix . 'security-signatureAlgorithm', null)
```

Comment:
Doe null fall back to the current default?

Reply:
Yes, that way we use the default from php-saml https://github.com/nextcloud/user_saml/blob/master/3rdparty/vendor/onelogin/php-saml/src/Saml2/Settings.php#L395-L397